dogfood/README.md

```diff
@@ -104,6 +104,13 @@ x
 You can `kubectl apply -f examples/<disruption.yaml>` for any `example/` disruption files.
 For gRPC disruption, you can follow these [detailed steps](../docs/grpc_disruption/demo_instructions.md).
 
+### Sending Metrics to Datadog
+
+For the purposes of testing disruptions/workflows, you should make sure that the datadog agent is properly installed
+on the cluster that the client and server are running on. 3 of the major disruptive resources properly send metrics
+to Datadog (CPU, Network, Disk). The client contains computation related to these disruptions and can be tested using
```

Comment:
>  3 of the major disruptive resources properly send metrics to Datadog (CPU, Network, Disk).

I dont understand this sentence

Reply:
Changing, let me know if its still confusing after the next PR.